src/lib.rs

```diff
@@ -186,6 +186,13 @@ pub trait FromArgs: Sized {
     /// command, treating each segment as space-separated. This is to be
     /// used in the output of `--help`, `--version`, and similar flags.
     fn from_args(command_name: &[&str], args: &[&str]) -> Result<Self, EarlyExit>;
+
+    /// Get a String with just the arg names, e.g., options, flags, subcommands, etc.
+    ///
+    /// The first argument `command_name` is the identifier for the current
+    /// command, treating each segment as space-separated. The second argument,
+    /// args, is the rest of the command line arguments.
+    fn to_args(command_name: &[&str], args: &[&str]) -> String;
```

Comment:
Why did you end up using a static method here? I imagined this more to be `fn to_args(&self) -> Vec<String>;`. This has two advantages:

* The arguments need to be fully parsed first.
* This would allow the caller to escape the arguments as necessary.

Reply:
Err, I mean:

```
fn to_args(&self) -> Vec<String> {
    vec![]
}
```

We need to have this be a default method for two reasons. First, adding a non-default method to a trait is not backwards compatible. Second, this defaults custom implementations to not return anything, which is more private.